src/main/webapp/index.html

```diff
@@ -108,8 +107,31 @@ <h2>Archlinux user</h2>
                 </p>
             </div>
         </div>
-        <div>Icons made by <a href="https://www.flaticon.com/authors/good-ware" title="Good Ware">Good Ware</a>, 
+        <div id="comments_title"><h1>Comments</h1></div>
+        <div class="comments_page">
+            <div class="comment_section">
+                <table id="com_table">
+                    <colgroup>
+                        <col style="width: 30%;">
+                        <col style="width: 70%;">
+                    </colgroup>
+                    <tbody>
+                        <tr>
+                            <th>Name</th>
+                            <th>Comment</th>
+                        </tr>
+                    </tbody>
+                </table>
+            </div>
+            <div class="comment_form">
+                <h1 style="color: white;">Com Frm</h1>
```

Comment:
This abbreviation can be confusing to viewers of your page. Can we change this to "Comment Form"?